src/questionnaireitem_system_focus.js

```diff
@@ -0,0 +1,58 @@
+/**
+A QuestionnaireItemSystem that stores how often and how long the focus was moved away from the page.
+
+Reports as array in milliseconds how long the focus was lost each time.
```

Comment:
Please document the result data structure.
Please document that `window` is used.
This is so far not done by any QuestionnaireItem.

Reply:
And that the eventlistener are attached as non-capturing.